scripts/build_docs.py

```diff
@@ -1,3 +1,7 @@
+from mathy_pydoc.__main__ import main
```

Comment:
It's not super important, but it would make me happy if it was abundantly clear (through the name) that this `main()` function is not our own `main()` function, for example by saying `from mathy_pydoc.__main__ import main as pydoc_main` or something like that.

Reply:
Good point.  I had done that the first time ;-P